Plugins/GitUIPluginInterfaces/GitConfigStatus.cs

```diff
@@ -0,0 +1,26 @@
+namespace GitUIPluginInterfaces
+{
+    /// <summary>
+    /// Status of running git config
+    /// https://git-scm.com/docs/git-config#_description This command will fail with non-
+    /// zero status upon error.Some exit codes are:
+    /// The section or key is invalid (ret= 1) this is returned when the key is not set also.
+    /// No section or name was provided (ret= 2).
+    /// The config file is invalid (ret=3).
+    /// The config file cannot be written (ret=4).
+    /// You try to unset an option which does not exist or you try to unset/set an
+    /// option for which multiple lines match(ret = 5).
+    /// You try to use an invalid regexp(ret = 6).
+    /// On success, the command returns the exit code 0.
+    /// </summary>
```

Comment:
The current form will render poorly in the intellisense. This one _should_ render much better, please check though:

```suggestion
    /// <summary>
    ///  Status of running git config https://git-scm.com/docs/git-config#_description.
    ///  This command will fail with non-zero status upon error.Some exit codes are:
    ///  <code>
    ///   The section or key is invalid (ret= 1) this is returned when the key is not set also.
    ///   No section or name was provided (ret= 2).
    ///   The config file is invalid (ret=3).
    ///   The config file cannot be written (ret=4).
    ///   You try to unset an option which does not exist or you try to unset/set an option for which multiple lines match (ret = 5).
    ///   You try to use an invalid regexp (ret = 6).
    ///   On success, the command returns the exit code 0.
    ///  </code>
    /// </summary>
```